docs/.vitepress/config.js

```diff
@@ -32,6 +32,10 @@ module.exports = {
       {
         text: 'Release Notes',
         link: 'https://github.com/vuejs/vitepress/releases'
+      },
+      {
+        text: 'Translations',
+        items: [{ text: '中文', link: 'https://cn-vitepress.netlify.app/' }]
```

Comment:
Can you remove this from this PR and coordinate with the people behind https://cn.vitejs.dev/ to follow the same practices? I noticed a few problems with your translation in the landing page already:

- cn-vitepress instead of Vitepress
- Wrong copyright notice

Reply:
/cc @ShenQingchuan @QC-L

Reply:
OK, thanks for your reminding and help. I have removed the redundant codes and contacted @ShenQingchuan @ShenQingchuan.